### DISCUSSION TITLE
New priors: spatial priors - ICAR, BYM, BYM2

### DISCUSSION THREAD

**Discussion by chenyangkang** [pymc-devs/pymc3#7502] [Ideas]
Hey community,

I'm proposing to include several new spatial priors which have been gaining attention in the recent years.

These include ICAR, BYM, and BYM2 prior. Those are intuitive structured priors for spatial neighboring regions modeling and inference, and have been used in some high-impact research (e.g. https://www.nature.com/articles/s41559-021-01442-y). You can find the formula and description here: https://mc-stan.org/users/documentation/case-studies/icar_stan.html.

I also used them in my previous research, and I feel like it could be potentially implemented in the package. If we agree on that, I will be happy to work on it.

More examples of implementation in R-INLA:
https://gkonstantinoudis.github.io/INLA/StrokeSheffield.html

**Comment by ricardoV94** [comment]
We already have ICAR I think? You can add the others to pymc-experimental where we tend to put newer things these days, without too much burden

**Comment by chenyangkang** [comment]
Oh yes I see there is ICAR. It probably wasn't there two years ago. I will try to work on the others mentioned. Thank you.
